### DISCUSSION TITLE
Override the Control + Q default keybinding?

### DISCUSSION THREAD

**Discussion by TheinThanHlan** [zed-industries/zed#19375] [Feature Requests]
While I am editing in Editor  ctrl + q exit . How can I override that key binding?

**Comment by autrioly** [comment]
an advance on this issue ?

**Comment by AlphaJack** [comment]
1. zed: open keymap
2. Search for "Q"
3. Delete the default entry

<img width="1186" height="148" alt="image" src="https://github.com/user-attachments/assets/92a7a3de-c9b2-471f-9d84-5b95387cb6cd" />
